src/ensembl/src/content/app/browser/browser-cog/BrowserCogList.test.tsx

```diff
@@ -14,10 +15,10 @@ describe('<BrowserCogList />', () => {
   const defaultProps = {
     browserActivated: true,
     browserCogList: 0,
-    browserCogTrackList: { 'track:gc': 491 },
+    browserCogTrackList: { 'track:gc': Math.random() },
```

Comment:
faker.random.number ? It will be an integer then, rather than a float between 0 and 1/